### PR TITLE
Added contribution guide

### PR DESCRIPTION
**Pull request by nitindermohan**
PR for merging contribution guide

Added files
- contribution guide overview
- code of conduct
- Git Hooks

### CODE REVIEW

File: content/docs/contribution-guide/contributing-guide.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+weight: 401
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+# Contributing Overview
```
Comment (giobart) [inline content/docs/contribution-guide/contributing-guide.md]:
```suggestion
```
This title can be removed otw Hugo shows it twice
image

File: content/docs/contribution-guide/contributing-guide.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+**Welcome to Oakestra!** We're excited that you're interested in contributing to our project. This guide provides an overview of how to get started as a contributor. Whether you’re fixing bugs, improving documentation, or developing new features, your contribution is valued and important.
+
+## Our Community Goals
+
+Oakestra is a welcoming, collaborative, and inclusive community. We want Oakestra to be a place where contributors feel empowered to share ideas, ask questions, and grow. All contributors are expected to follow our ~Code of Conduct~ to keep our community positive and open.
```
Comment (giobart) [inline content/docs/contribution-guide/contributing-guide.md]:
```suggestion
Oakestra is a welcoming, collaborative, and inclusive community. We want Oakestra to be a place where contributors feel empowered to share ideas, ask questions, and grow. All contributors are expected to follow our _Code of Conduct_ to keep our community positive and open.
```

I guess you meant to underline the text?

File: content/docs/contribution-guide/contributing-guide.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Contributors and people new to open source alike will find the following guides especially useful.
+
+{{< card-grid >}}
+{{< link-card description="How to Contribute to Open Source" href="https://opensource.guide/how-to-contribute/" target="_blank" >}}
+{{< link-card description="Building Welcoming Communities" href="https://opensource.guide/building-community/" target="_blank" >}}
+{{< /card-grid >}}
+
+## Code of Conduct
+
+Oakestra is committed to a respectful and inclusive environment. By contributing, you agree to abide by our [Code of Conduct](#link-to-code-of-conduct).
```
Comment (giobart) [inline content/docs/contribution-guide/contributing-guide.md]:
```suggestion
Oakestra is committed to a respectful and inclusive environment. By contributing, you agree to abide by our [Code of Conduct](../code-of-conduct/).
```

File: content/docs/contribution-guide/contributing-guide.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+## How You Can Contribute
+
+If you are eager to start contributing code right away, we have a list of [good first issues](https://github.com/oakestra/oakestra/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. Alternatively, you can ceck for issues labeled as [“help wanted”](https://github.com/oakestra/oakestra/labels/help%20wanted) for beginner-friendly feature extensions. As you gain more experience and demonstrate a commitment to evolving Oakestra, you may be granted issue management permissions in the repository.
```
Comment (giobart) [inline content/docs/contribution-guide/contributing-guide.md]: good first issues are usually the beginner-friendly ones, while help wanted all the rest. Also we need to specify how they can get the issue assigned. Here a suggestion:

```suggestion
If you are eager to start contributing code right away, we have a list of [good first issues](https://github.com/oakestra/oakestra/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. Alternatively, you can ceck for issues labeled as [“help wanted”](https://github.com/oakestra/oakestra/labels/help%20wanted) for feature extensions that have not been handled yet. As you gain more experience and demonstrate a commitment to evolving Oakestra, you may be granted issue management permissions in the repository.

**How to contribute?** Just comment on an unassigned issue, stating you want to work on it. Once you get an ACK, you'll get assigned to a branch related to the issue. All your work must be merged to the assigned branch first via Pull Request. 
```

File: hugo_stats.json
Comment (giobart) [inline hugo_stats.json:1]: this file currently has a conflict

**Review by giobart — CHANGES_REQUESTED** [review]
Super cool! Just added my review comments :D

File: hugo_stats.json
Comment (nitindermohan) [reply hugo_stats.json:1]: Should be fine now.

**Comment by nitindermohan** [comment]
@giobart thank you for your comments. I also just added the code style guide doc to the PR. Can you please review it as well?

**Review by giobart — COMMENTED** [review]
Everything looks veery good. There is only one issue with the PyCharm/VSCode tables (at least on safari) but I think this issue is related to Doks. Specifically, when clicking PyCharm/VSCode the page jumps to a different VSCode/PyCharm table in the same page. As you see the tables are actually switching but the page anchors are messed up. I think that if there is no immediate solution we can ignore this for now.
ScreenRecording2024-12-11at16 16 43-ezgif com-video-to-gif-converter

**Comment by nitindermohan** [comment]
Discussed the concern pointed out by @giobart (in comment https://github.com/nitindermohan/oak-new-docs/pull/25#pullrequestreview-2496012361) in the meeting and it appears like a browser-specific issue of Doks. The bug appears in Safari but not in Chrome. 

Doks is planning to make a new release soon and hopefully this will be addressed with it.

**Pull request merged**